### PR TITLE
Fix copy config file issue

### PR DESCRIPTION
**Pull request by mbilal92**

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ core:
 
 ueransim:
   servers:
-    0:
-      - "config/custom-gnb.yaml"
-      - "config/custom-ue.yaml"
+    "0":
```
Comment (gab-arrobo) [inline vars/main.yml:9]: Why does this have to be a string?

File: roles/simulator/defaults/main.yml [hunk 2]
```diff
@@ -5,8 +5,9 @@ ueransim:
         name: ueransimnet
 
   servers:
-    0:
-      - "config/ueransim_default.yaml"
+    "0":
```
Comment (gab-arrobo) [inline roles/simulator/defaults/main.yml:8]: Why does this have to be a string?

File: roles/simulator/tasks/start.yml [hunk 3]
```diff
@@ -1,38 +1,43 @@
...
+    server_index: "{{ groups['ueransim_nodes'].index(inventory_hostname) }}"  
+  when: inventory_hostname in groups['ueransim_nodes']
+
+- name: copy UE config files to ueransim host
+  template:
+    src: "{{ ROOT_DIR }}/{{ ueransim.servers[server_index].ue }}"
+    dest: ueransim_onramp/config/custom-ue.yaml
+  vars:
+    server_index: "{{ groups['ueransim_nodes'].index(inventory_hostname) }}"  
   when: inventory_hostname in groups['ueransim_nodes']
```
Comment (gab-arrobo) [inline roles/simulator/tasks/start.yml:22]: FYI, as it was before, there was no need to have 2 tasks (one for the `ue` and one for the `gnb`). Do you have to create 2 tasks? or is it to make easier to visualize?

Comment (mbilal92) [reply roles/simulator/tasks/start.yml:22]: Initially, the path was set to `config/filename`, but in OnRamp, we follow the convention `deps/ueransim/config/filename`. It wasn't clear which file was for the UE and which was for the gNb. To resolve this, I added tags and now copy each file, renaming them accordingly (for UE and gNb) in a separate loop.

File: vars/main.yml (hunk 1 above)
Comment (mbilal92) [reply vars/main.yml:9]: It makes it easier to loop through and find the corresponding server.

Comment (gab-arrobo) [reply vars/main.yml:9]: Then, why do you use "numbers" in the `gnbsim` repo (https://github.com/gab-arrobo/aether-gnbsim/blob/master/vars/main.yml#L18)? I was thinking that these two (gnbsim, and ueransim) should be similar (preferable, numbers)

File: roles/simulator/tasks/start.yml (hunk 3 above)
Comment (gab-arrobo) [reply roles/simulator/tasks/start.yml:22]: Ok, thanks!

**Pull request merged**